src/components/queue/QueueListItem/QueueListItem.tsx

```diff
@@ -120,7 +120,7 @@ const QueueListItem: FC<QueueListItemProps> = ({queue, ticket, position}) => {
                                         </Typography>
                                         {queue.showMeetingLinks && claimedUser.meetingLink && (isTicketOwner || isTA || currentUser?.isAdmin) && (
                                             <Button sx={{display: "inline-flex", alignItems: "center"}}
-                                                    href={claimedUser?.meetingLink} variant="text" color="inherit"
```

Comment:
This component has the expected behavior from before (command clicking the link will open in a new tab). Please use `target="_blank" rel="noopener noreferrer"` instead of `wndow.open` to open in a new tab.